src/openepd/api/base_sync_client.py

```diff
@@ -159,7 +160,7 @@ def __init__(
         requests_per_sec: float = 10,
         retry_count: int = 3,
         user_agent: str | None = None,
-        timeout_sec: float | tuple[float, float] | None = None,
+        timeout_sec: float | tuple[float, float] | None = DEFAULT_TIMEOUT_SEC,
```

Comment:
`self.timeout = timeout_sec or DEFAULT_TIMEOUT_SEC` - this approach will allow you not to change constructor, as a result, inheritors will not need to change constructor as well and import `DEFAULT_TIMEOUT_SEC`.